### DISCUSSION TITLE
Enhancement: Separate the enemy vocals and bf vocals in the files

### DISCUSSION THREAD

**Discussion by Jmanx7** [FunkinCrew/Funkin#1583] [Ideas]
If the two parts were better distinguished, duets would no longer cut out for the enemy when the player misses a note. This would also make modding songs with duets easier if the modder doesn't want to change one part of the duet.

**Comment by Templarfreak** [comment]
This is a good idea and probably would not be a hard code change to make, the most painful part would probably be recomposing all the voice track files to separate the two voices into two files. In my opinion the game is starting to need little polish details like this.

**Comment by regulad** [comment]
I'm sure it would also help people making mods that put in remixes.
